.circleci/config.yml

```diff
@@ -16,4 +16,4 @@ jobs:
             - ./node_modules
       - run:
           name: lint
-          command: npm run lint
```

Comment:
This change here breaks circleci. The part `command: npm run lint` is in the wrong logic block. The file is a yaml file. Just remove this change and things will work again.